RadarSDKTests/RadarSDKTests.m

```diff
@@ -1489,5 +1489,47 @@ - (void)test_RadarLogBuffer_purge {
     [[RadarLogBuffer sharedInstance]clearBuffer];
 }
 
+- (void)test_RadarSdkConfiguration {
```

Comment:
I feel that having a test using initialize might be appropriate here to give better guarantees about e2e behaviors.  Granted we need to mock a bunch of stuff...

Reply:
I added userId and metadata check for initialize, It's difficult to verify if trackOnce was called on init, I feel like the api.getConfigForUsage is good enough to cover the same call from init (verified if the variable is set).